packages/core/build/config.js

```diff
@@ -18,7 +18,7 @@ const copyConfig = base => {
             },
             {
                 from: path.resolve(__dirname, '../node_modules/@deriv/dashboard/lib/assets/images'),
-                to: 'public/images',
+                to: 'js/dashboard/assets/images',
```

Comment:
it might be because of this , pls revert this change 🙏

Reply:
This is what Mamali had changed before and you asked me to revert to `'js/dashboard/assets/images'` .This is like what we have on master now.